scripts/mutation-test.sh

```diff
@@ -1,35 +1,51 @@
 #!/usr/bin/env bash
 
 set -eo pipefail
+oIFS="$IFS"; IFS=, ; set -- $1 ; IFS="$oIFS"
 
 DISABLED_MUTATORS='branch/*'
 
 # Only consider the following:
-# * go files in types or keeper packages
+# * go files in types, keeper, or module root directories
 # * ignore test and Protobuf files
 MUTATION_SOURCES=$(find ./x -type f \( -path '*/keeper/*' -or -path '*/types/*' \) \( -name '*.go' -and -not -name '*_test.go' -and -not -name '*pb*' \))
+MUTATION_SOURCES+=$(find ./x -maxdepth 2 -type f \( -name '*.go' -and -not -name '*_test.go' -and -not -name '*pb*' \))
 
-# XXX: Filter on a module-by-module basis and expand when we think other modules
-# are ready. Once all modules are considered stable enough to be tested, remove
-# this filter entirely.
-MUTATION_SOURCES=$(echo "$MUTATION_SOURCES" | grep './x/tokenfactory')
+# Filter on a module-by-module basis as provided by input
+arg_len=$#
+
+for i in "$@"; do
+  if [ $arg_len -gt 1 ]; then
+    MODULE_FORMAT+="./x/$i\|"
+    MODULE_NAMES+="${i} "
+    let "arg_len--"
+  else
+    MODULE_FORMAT+="./x/$i"
+    MODULE_NAMES+="${i}"
+  fi
+done
+
+MUTATION_SOURCES=$(echo "$MUTATION_SOURCES" | grep "$MODULE_FORMAT")
 
 # Collect multiple lines into a single line to be fed into go-mutesting
 MUTATION_SOURCES=$(echo $MUTATION_SOURCES | tr '\n' ' ')
+echo $MUTATION_SOURCES
 
-OUTPUT=$(go run github.com/osmosis-labs/go-mutesting/cmd/go-mutesting --disable=$DISABLED_MUTATORS $MUTATION_SOURCES)
+# echo "running mutation tests for the following module(s): $MODULE_NAMES"
+# OUTPUT=$(go run github.com/osmosis-labs/go-mutesting/cmd/go-mutesting --disable=$DISABLED_MUTATORS $MUTATION_SOURCES)
 
-# Fetch the final result output and the overall mutation testing score
-RESULT=$(echo "$OUTPUT" | grep 'The mutation score')
-SCORE=$(echo "$RESULT" | grep -Eo '[[:digit:]]\.[[:digit:]]+')
+# # Fetch the final result output and the overall mutation testing score
+# RESULT=$(echo "$OUTPUT" | grep 'The mutation score')
+# SCORE=$(echo "$RESULT" | grep -Eo '[[:digit:]]\.[[:digit:]]+')
 
-echo "writing mutation test result to mutation_test_result.txt"
-echo "$OUTPUT" > mutation_test_result.txt
+# echo "writing mutation test result to mutation_test_result.txt"
```

Comment:
are we not writing the results to .txt file anymore?

Reply:
Was no meant to be commented out, thanks for pointing out!